pkg/koordlet/metricsadvisor/collector.go

```diff
@@ -124,6 +125,7 @@ func (c *collector) Run(stopCh <-chan struct{}) error {
 	}
 
 	go wait.Until(func() {
+		c.collectGPUUsage()
```

Comment:
how about consider enable gpu collector *only if* node has gpu.
besides, please add a new config for this function since it's a new feature.

Reply:
If there is no GPU or driver, collectGPUUsage will do nothing.

Reply:
@LambdaHJ It does invoke `nvml.Init()` (where CGO is required & tries to initialize NVML every time) and print a WARNING msg. Could you please make it optional?